snuba/admin/tool_policies.py

```diff
@@ -60,6 +64,15 @@ def check_tool_perms(
     def decorator(f: Callable[..., Response]) -> Callable[..., Response]:
         @wraps(f)
         def check_perms(*args: Any, **kwargs: Any) -> Response:
+            if settings.ADMIN_DEVELOPER_MODE:
+                for tool in tools:
+                    if tool in DEVELOPER_TOOLS:
```

Comment:
Looks good, only question I have is what happens if an admin endpoint requires two [different roles](https://github.com/getsentry/snuba/blob/76420cf365a7429333fd31f1a5f730e462c2494b/snuba/admin/views.py#L675). Should we check all required tools exist in `DEVELOPER_TOOLS` set?

Reply:
No, that filter is an "OR" filter. So people with access to the system queries tool OR the tracing tool can access that endpoint.